contracts/TitleEscrow.sol

```diff
@@ -111,24 +115,51 @@ contract TitleEscrow is Context, ITitleEscrow, HasNamedBeneficiary, HasHolder, E
     emit TransferEndorsed(_tokenId, beneficiary, newBeneficiary);
     approvedTransferTarget = newBeneficiary;
   }
-
-  function transferTo(address newBeneficiary) public isHoldingToken onlyHolder allowTransfer(newBeneficiary) {
+  
+  function _transferTo(address newOwner) private {
```

Comment:
what's the benefit to have this private function ?

Reply:
this is an internal function that can only be called by this instance of the contract, and should never be exposed to external parties since it is the function that actually execute the transfer. It assumed that checks have been performed, so it does the transfer without additional checks. It is only to be called from other functions in this contract.